Jenkinsfile

```diff
@@ -31,3 +42,37 @@ node('linux') {
 			sh 'make test'
 	}
 }
+
+def git_commit = ''
+def go_list = ''
+
+stage 'Go & Git Info'
+node('linux') {
+
+	def gopath = gopath_linux()
+	def workdir = workdir_linux(gopath)
+
+	dir(workdir) {
+		checkout scm
+
+		// HACK: Recommended approach for getting command output is writing to and then reading a file.
+		sh 'mkdir -p tmp'
+		sh 'go list . > tmp/GO_LIST'
```

Comment:
@arschles Jenkins Pipeline has its own PR logic (with much less functionality of the PR builder plugin).  As far as I can tell, there doesn't seem to be a way to include the plugin in a Jenkinsfile (https://issues.jenkins-ci.org/browse/JENKINS-32556) but definitely worth investigating in a different ticket.  

Here is the `env` output from this PR Pipeline: https://gist.github.com/vdice/945ea887eba1059c6282ec2922adc889

It appears the info we're limited to at this point (unless we can get at it via `git` commands on the shell, etc.) are `CHANGE_URL`, `CHANGE_TITLE`, `CHANGE_AUTHOR`, `CHANGE_ID` and `CHANGE_TARGET`.